docs/project_ram/RAM_1pager.md

```diff
@@ -0,0 +1,41 @@
+# Research Application Management
+## Developing research outputs for real-world problem solving while ensuring openness and reproducibility
+
+Research Application Managers (RAMs) focus on the sustainability and impact of research, guiding the development and use of research outputs for a broad set of stakeholders. The [role of a RAM](https://the-turing-way.netlify.app/collaboration/research-infrastructure-roles/ram.html) was created at the Turing to help fill the gaps between purely academic knowledge generation and the work that is required to create and maintain user-friendly tools that can be used to address questions beyond the scope of the original research field.
+
+![image](https://github.com/alan-turing-institute/research-application-management/assets/5104098/108b7fc9-4b1a-4325-964f-b19561e89f28)
+
+## What skills do RAMs bring to research teams?
+
+RAMs think [beyond the research project cycle](https://journal.trialanderror.org/pub/manifesto-rewarding-recognizing/release/1), creating opportunities to reuse and reproduce research outputs in new real-world applications. At the Turing, RAMs bring product development and stakeholder engagement skills to research teams and cultivate a practice of open collaboration, encouraging interconnectedness of research within the Institute and beyond.
+
+As a team, we develop and deliver a range of stakeholder-related activities including:
+* Stakeholder mapping
+* Requirements gathering & alignment
+* User-centred design
+* Research output development roadmapping
+* Workshop delivery
+
+Learn more about our work and access our tools and insights [here](https://the-turing-way.netlify.app/collaboration/stakeholder-engagement.html).
+
+## What makes for an impactful RAM engagement?
+
+We consider multiple dimensions of a research project and team to prioritise engagement across Turing's research projects:
+* **Applicability**: How much potential does a research project have for real-world use?
+* **Capability**: To what extent does the team have the skills to translate their research to real-world use?
+* **Values**: How much does the team's approach to research align with our focus areas (e.g. interoperable open source tools, interdisciplinary collaboration, participatory co-creation)
+* **Practices**: How ready are the team to adopt RAM ways of working?
+ 
+![RAM rhombus-Simple diagram drawio](https://github.com/alan-turing-institute/research-application-management/assets/5104098/19d17aff-5caa-441f-8703-1497553b1ab3)
```

Comment:
We should maybe rename this to something more slick? Idk

Reply:
And maybe highlight in the diagram the different levels (high for applicability, practices, values, low for capabilities).

We should maybe also source where we got that idea from, though that might be for a longer report

Reply:
Yep, agree on expanding on the [INSERT SNAZZY NAME HERE] on a longer paper/report that we can link to here!